src/components/molecules/Navbar/index.jsx

```diff
@@ -59,136 +64,146 @@ class Navbar extends React.Component {
+                  <div className="spacer" />
+                  <MDBNavItem>
+                    <MDBDropdown>
+                      <MDBDropdownToggle nav caret>
+                        <img
+                          src={avatarUrl}
+                          className="z-depth-0"
+                          alt={loggedUser.username}
+                        />
+                      </MDBDropdownToggle>
+                      <MDBDropdownMenu className="dropdown-default">
+                        <Link to="/settings" className="dropdown-item">
+                          Settings
+                        </Link>
+                        <Link
+                          to="/"
+                          onClick={this.props.logout}
+                          className="dropdown-item"
+                        >
+                          Sign Out
+                        </Link>
+                      </MDBDropdownMenu>
+                    </MDBDropdown>
+                  </MDBNavItem>
+                </>
+              ) : (
+                <>
+                  {location.pathname !== "/" && (
+                    <Link
+                      to={{
+                        pathname: "/",
+                        state: {
+                          actionCard: 1,
+                        },
+                      }}
+                    >
+                      <MDBBtn color="green" outline size="md">
                         Sign In
                       </MDBBtn>
-                    )}
-                  </>
-                )}
-              </MDBNavbarNav>
-            </MDBCollapse>
-          </MDBContainer>
-        </MDBNavbar>
-        {this.state.showSettings && (
-          <Suspense fallback={<div>Loading...</div>}>
-            <Settings
-              {...this.props}
-              closeModal={this.handleSettingsClose}
-              saveSettings={globalFunctions.saveSettings}
-            />
-          </Suspense>
-        )}
-      </>
+                    </Link>
+                  )}
+                </>
+              )}
+            </MDBNavbarNav>
+          </MDBCollapse>
+        </MDBContainer>
+      </MDBNavbar>
     );
   }
 }
 //#endregion
 
 //#region > PropTypes
 Navbar.propTypes = {
-  globalState: PropTypes.object,
-  globalFunctions: PropTypes.object,
   location: PropTypes.object,
 };
 //#endregion
 
+//#region > Redux Mapping
+const mapStateToProps = (state) => ({
+  loggedUser: { ...state.auth.loggedUser, ...state.user.loggedUser },
+});
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    logout: () => dispatch(logoutAction()),
+  };
+};
+//#endregion
+
 //#region > Exports
-//> Default Class
-export default withRouter(Navbar);
+/**
+ * Provides its connected component with the pieces of the data it needs from
+ * the store, and the functions it can use to dispatch actions to the store.
+ * Got access to the history object’s properties and the closest
+ * <Route>'s match.
+ */
```

Comment:
```suggestion
/**
 * Provides its connected component with the pieces of the data it needs from
 * the store, and the functions it can use to dispatch actions to the store.
 *
 * Got access to the history object’s properties and the closest
 * <Route>'s match.
 */
```